apps/vmq_server/src/vmq_schema.erl

```diff
@@ -497,6 +504,7 @@ translate_listeners(Conf) ->
             HTTP_SSLConfigMod,
             HTTP_SSLConfigFun,
             HTTP_SSLMaxLengths,
+            HTTP_SSLMaxHeaderLengths
```

Comment:
`,` missing between variables. Please also run the code formatter with `rebar3 fmt` before a final commit. Thanks!